content/12.privacy.md

```diff
@@ -1 +1,11 @@
 ## Rule 10: Don't share models trained on sensitive data
+
+One of the greatest opportunities for deep learning in biology is the ability for deep learning techniques to incorporate representation learning to extract information that can not readily be captured by traditional methods [@arxiv:1509.09292]. 
+The abundance of features for each sample means that the models can learn an enormous amount about the data during the training process, and therefore it is important not to share models trained on sensitive data. 
```

Comment:
Does sample refer to the training set or each instance in the training set? I guess both would be appropriate, but  maybe we should stick to machine learning terminology and use "example" instead of "sample" when we mean individual instances. 

"models can learn an enormous amount about" => I think it's not necessarily learning about/from the data but that the representation learning in deep learning kind of captures information-rich abstractions of the data in the hidden layers. Maybe also worth mentioning KNN as an extreme example (literally stores training data) to indicate that not all traditional models are immune of this problem.

Reply:
Yes, good catch. The biologist in me thinks everything in life is just a "sample".

Reply:
true, as a bio/stats person I also use(d) the term sample usually as well. Even throughout my ML book. But I noticed that it can confuse people because in ML contexts, it's not clear whether it refers to the whole training set. I think instead of "example" here "training example" would be even better.

Reply:
Great point, I just changed it to "training example" to be most consistent with ML literature.

Reply:
awesome, thx!